Libplanet.Net.Tests/Consensus/ContextNonProposerTest.cs

```diff
@@ -228,7 +228,7 @@ public async Task EnterPreVoteNilOnInvalidBlockHeader()
             var invalidBlock = blockChain.EvaluateAndSign(
                 new BlockContent(
                     new BlockMetadata(
-                        protocolVersion: BlockMetadata.CurrentProtocolVersion - 1,
+                        protocolVersion: BlockMetadata.CurrentProtocolVersion,
```

Comment:
I think this test is a bit weird...
Generating multiple invalid blocks with different condition would be proper instead of generating single block here.
Though, `BlockMetadata.CurrentProtocolVersion - 1` intend to generate invalid block, so I think we don't need to change protocol version here for now.

Reply:
I've changed this since the block became to fail on evaluation process if the protocol version mismatches. Was failed on append process before.

Reply:
👌